docs/CHANGELOG.rst

```diff
@@ -6,6 +6,15 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
+Unreleased
+==========
+
+Changed
+-------
+- "Add ``username`` to ``get_object_perms`` shortcut
```

Comment:
Please describe a user-facing change, not the implementation details, e.g. `Add username to permission objects on the API`. And move it under `Added`.

Reply:
Done.

Reply:
If this fix makes it OK, please merge.